exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/aggregate/HashAggTemplate.java

```diff
@@ -84,13 +85,6 @@
 public abstract class HashAggTemplate implements HashAggregator {
   protected static final org.slf4j.Logger logger = org.slf4j.LoggerFactory.getLogger(HashAggregator.class);
 
-  private static final int VARIABLE_MAX_WIDTH_VALUE_SIZE = 50;
-  private static final int VARIABLE_MIN_WIDTH_VALUE_SIZE = 8;
-
-  private static final boolean EXTRA_DEBUG_1 = false;
```

Comment:
This is Aman's implementation of an #ifdef .....
Basically he wanted to preserve these debugging messages, but not seeing them (there are too many) while doing some other debugging work.

Reply:
Logging frameworks were invented to solve this problem. We should just use the logging framework.

Reply:
Oh but there is! slf4j and logback have a feature called markers, which allows you to associate a tag with a statement. When you print logs you can specify to filter by level and by marker. There is a working example here https://examples.javacodegeeks.com/enterprise-java/slf4j/slf4j-markers-example/ . I will update the log statements to use markers in this PR.